lib/crawley/local_storage.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Crawley.LocalStorage do
+  @behaviour Crawley.Storage
+
+  @impl Crawley.Storage
+  def upload_repo(remote_folder_name, repo_name) do
+    zip_folder(repo_name)
+
+    File.rename(
+      "./tmp/#{repo_name}/#{repo_name}.zip",
+      "./tmp/buckets/#{remote_folder_name}/#{repo_name}.zip"
+    )
+  end
+
+  defp zip_folder(repo_name) do
```

Comment:
Is there any way to move this code to the Storage definition? Because it's duplicated with the code on `Crawley.S3`